### PR TITLE
Badge docs

### PR DESCRIPTION
**Pull request by el-dav**

### CODE REVIEW

File: site/docs/components/badge/accessibility.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Badge
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+We recommend only using the accent color for Badge. Align to the minimum contrast guidance outlined in [WCAG 2.1](<(https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)>) for any color overrides.
```
Comment (origami-z) [inline site/docs/components/badge/accessibility.mdx:10]: WCAG link in preview is invalid

File: site/docs/components/badge/examples.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  </LivePreview>
+  <LivePreview componentName="badge" exampleName="InlineBadge">
+
+    ### Inline Badge
+
+    To place a badge inline, instead of in the top-right corner of an element simply do not pass `children` to Badge.
+
+    #### Guidance
+
+    Using badge inline with text makes clear which part of the content the badge relates to and allows specific details to be highlighted within the content to draw the user’s attention. For example, you may use badge within a tab, inline to the text.
```
Comment (libertymayc) [inline site/docs/components/badge/examples.mdx:31]:
```suggestion
    Using Badge inline with text makes clear which part of the content the badge relates to and allows specific details to be highlighted within the content to draw the user’s attention. For example, you may use Badge within Tab, inline to the text.
```

File: site/docs/components/badge/usage.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+### When to use Badge
+
+- To communicate a non-interactive, system driven change to the underlying component it is anchored to. For example, in tabs, pages and containers.
+
+- Typically, if a badge is used to indicate an unread notification, the badge is hidden once the notification is addressed. However, badges can remain persistent depending on context.
+
+### When not to use Badge
+
+- To organize content within categories, trigger an immediate action, or enable an action that may change depending on context. Instead, use [Pill](/salt/components/pill).
```
Comment (libertymayc) [inline site/docs/components/badge/usage.mdx:22]:
```suggestion
- To organize content within categories, trigger an immediate action, or enable an action that may change depending on context. Instead, use [Pill](../pill).
```

File: site/docs/components/badge/usage.mdx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+- To communicate status through color, such as RAG statuses. Instead, use Pill. Badges are independent of user action and therefore should not be used to communicate something the user can directly influence through a task.
+
+### Import
+
+To import badge from the lab Salt package, use:
```
Comment (lilyvc) [inline site/docs/components/badge/usage.mdx:28]:
```suggestion
To import Badge from the lab Salt package, use:
```

**Review by alycrys — APPROVED** [review]

**Pull request merged**